pkg/retry/azure_error.go

```diff
@@ -425,3 +425,16 @@ func getOperationNotAllowedReason(msg string) string {
 	}
 	return OperationNotAllowed
 }
+
+// AcceptedError implements error interface. It is meant to be returned for errors with http status code of 2xx
+type AcceptedError struct {
```

Comment:
While technically correct, `AcceptedError` is a confusing name. Consider `OperationPreemptedError`. Perhaps `PartialUpdateError` if you want the flexibility to extend it to codes other than `OperationPreempted` in we need to in the future?